org.ghidra_sre.Ghidra.json

```diff
@@ -50,9 +50,9 @@
             "build-commands": [
                 "for i in docs Extensions Ghidra GPL licenses server support ; do install -d $i /app/lib/ghidra/ ; cp -r $i/ /app/lib/ghidra/$i/ ; done",
                 "install -Dm755 ghidraRun /app/lib/ghidra/ghidraRun",
-                "icotool -x support/ghidra.ico --index=8",
-                "install -Dm644 ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Ghidra.png",
+                "icotool -x support/ghidra.ico -i 8",
```

Comment:
Why is this "improved"? Doesn't seem to be any different, and I prefer longer options as that means I don't need to look up the meaning of argument.

Reply:
Right. I didn't thought that longer argument could help for a better understanding. I'll correct that.

Reply:
Corrected